### ISSUE TITLE
"The actor is blocking and will not use any CPU if there are no messages to it."

### ISSUE THREAD

**Issue by scheung38** [krupio/ActorLite#1]
"The actor is blocking and will not use any CPU if there are no messages to it."

so it is not truly asynchronised? But I thought Actors is to be non-blocking as originally intended?

Seb

**Comment by krupio** [comment]
No, they are asynchronous. They just sleep if there is nothing for them to do i.e. there is no message for them. That what I meant when I wrote "blocking"

**Comment by scheung38** [comment]
Thanks I will learn how to use it. So how different is it as compared to using multiprocessing module?
